compiler+runtime/bin/ci/validate-formatting

```diff
@@ -3,7 +3,11 @@
 set -euo pipefail
 
 success=true
-clang_format="./build/llvm-install/usr/local/bin/clang-format"
+if [[ -e "./build/llvm-install" ]]; then
+    clang_format="./build/llvm-install/usr/local/bin/clang-format"
+else
+    clang_format="$(command -v clang-format)"
+fi
```

Comment:
Please drop the `then` down, Allman style. Also, this file has 2 space indentation.

Reply:
Worth adding an editorconfig file, I can add also if you want

Reply:
Up to you if it's worth it.